apps/commudle-admin/src/app/app-shared-components/event-card/event-card.component.html

```diff
@@ -24,7 +24,14 @@
                 <nb-icon icon="checkmark-circle-2" nbTooltip="Completed" status="success" class="com-m-auto"></nb-icon>
               </div>
               <div *ngIf="event.interested_members_count > 0" class="interested-members">
-                <span>Interested Members: {{ event.interested_members_count }}</span>
+                <app-badge
+                  [text]="event.interested_members_count"
+                  [color]="'#F0F2F8'"
```

Comment:
Is there a way to pass the color name and then use it from tailwind in the app-badge component?